uk.co.compucorp.civicrm.hrcomments/info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.hrcomments" type="module">
+  <file>hrcomments</file>
+  <name>Comments</name>
+  <description>Track user comments</description>
```

Comment:
I think we can elaborate this a bit more. What about using the same sentence from the README file? ("This extension provides Commenting functionality for CiviHR")